modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -124,6 +125,64 @@ PyObject* pyopencv_from(const cv::detail::PyObjectHolder& v)
     return o;
 }
 
+// #FIXME: Is it possible to implement pyopencv_from/pyopencv_to for generic
+// cv::variant<Types...> ?
+template <>
+PyObject* pyopencv_from(const cv::gapi::wip::draw::Prim& prim)
+{
+    switch (prim.index()) {
```

Comment:
I believe it's possible to implement `pyopencv_from` for generic `variant` by using `visitor`, but it's not merged yet.